src/NuGet.Clients/NuGet.CommandLine/Commands/ProjectFactory.cs

```diff
@@ -290,6 +300,18 @@ public Packaging.PackageBuilder CreateBuilder(string basePath, NuGetVersion vers
                         string.Format(NuGetResources.ProjectJsonPack_Deprecated, builder.Id),
                         NuGetLogCode.NU5126));
                 _usingJsonFile = true;
+
+                _ = bool.TryParse(_environmentVariableReader.GetEnvironmentVariable(NUGET_ENABLE_LEGACY_PROJECT_JSON_PACK), out bool enableLegacyProjectJsonPack);
+
+                if (!enableLegacyProjectJsonPack)
+                {
+                    Logger.Log(
+                        PackagingLogMessage.CreateError(
+                            string.Format(NuGetResources.Error_ProjectJson_Deprecated_And_Removed, builder.Id, NUGET_ENABLE_LEGACY_PROJECT_JSON_PACK),
+                            NuGetLogCode.NU5041));
+                    return null;
```

Comment:
Please include screenshot or text of this log from product.

Reply:
Not sure what you mean you can refer to the resource. 

> `project.json` pack is disabled in the current NuGet version, and will be permanently removed in a future version.
Please consider migrating 'project1' to `PackageReference` and using the pack targets.
You can use 'NUGET_ENABLE_LEGACY_PROJECT_JSON_PACK' environment variable to temporarily reenable this functionality.